### PR TITLE
Featured frameworks

### PR DESCRIPTION
**Pull request by chadwcarlson**
Extends https://github.com/platformsh/platformsh-docs/pull/1196, which will probably be marked as a duplicate and closed.

### CODE REVIEW

File: naming.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+@devrel_team - some follow-up on our Service Naming Convention conversation we had a few weeks ago. Feedback welcome.
```
Comment (Crell) [inline naming.md]: Does this file really belong in the public repo?  I think very no.  Just the final outcome, in abbridged form, goes in CONTRIBUTING.md.  I wouldn't even have this in the history, so please rebase.

File: src/frameworks/hibernate.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+## Services
+
+### MySQL, MariaDB and PostgreSQL
+
+[MySQL, MariaDB](configuration/services/mysql.md), and [PostgreSQL](configuration/services/postgresql.md) are open-source relational database technologies.
+
+The first step is to choose the database that you would like to use in your project. Define the driver for [MySQL](https://mvnrepository.com/artifact/mysql/mysql-connector-java), [MariaDB](https://mvnrepository.com/artifact/org.mariadb.jdbc/mariadb-java-client) or [PostgreSQL](https://mvnrepository.com/artifact/postgresql/postgresql), and the Java dependencies. Then determine the SessionFactory client programmatically:
+
+#### MySQL
```
Comment (Crell) [inline src/frameworks/hibernate.md]: We've managed to avoid a 4th level header so far, I think.  Let's keep it that way.

Comment (Crell) [reply src/frameworks/hibernate.md]: Probably just break these 3 out to their own sections and eliminate the common header.

File: src/frameworks/jakarta.md [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+    }
+
+    public void destroy(@Disposes BucketManager manager) {
+        manager.close();
+    }
+
+}
+```
+
+### MySQL, MariaDB and PostgreSQL
```
Comment (Crell) [inline src/frameworks/jakarta.md]: Ibid.

File: src/frameworks/spring.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+        final RedisTemplate<String, Object> template = new RedisTemplate<String, Object>();
+        template.setConnectionFactory(jedisConnectionFactory());
+        template.setValueSerializer(new GenericToStringSerializer<Object>(Object.class));
+        return template;
+    }
+
+}
+```
+
+### MySQL, MariaDB and PostgreSQL
```
Comment (Crell) [inline src/frameworks/spring.md]: Ibid.

File: src/languages/php/frameworks.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
+# PHP Featured Frameworks
+
+Platform.sh is the chosen cloud solution behind Symfony, Magento, eZ Systems, and Drupal Commerce. Full Drush support and Composer-based builds make handling dependencies and builds for PHP frameworks as simple as committing your `composer.json` to your project.
```
Comment (Crell) [inline src/languages/php/frameworks.md]: I'm not sure that we're allowed to say that publicly. :smile:

File: src/languages/php/frameworks.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+Drupal FAQs, how-to guides and other tutorials right on [Platform.sh Community](https://community.platform.sh/).
+
+* [Drupal on Platform.sh Community](https://community.platform.sh/search?q=drupal&expanded=true)
+
+### Example Projects
+
+Check out the fully configured Drupal starter projects maintained by the Platform.sh team.
+
+#### Templates
```
Comment (Crell) [inline src/languages/php/frameworks.md]: Flatten the hierarchy, please.

File: src/languages/php/frameworks.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
+# PHP Featured Frameworks
+
+Platform.sh is the chosen cloud solution behind Symfony, Magento, eZ Systems, and Drupal Commerce. Full Drush support and Composer-based builds make handling dependencies and builds for PHP frameworks as simple as committing your `composer.json` to your project.
+
+## Drupal
+
+Drupal 8 is built around two technologies that make developer's lives easier: Composer and Symfony. Composer simplifies dependency management for PHP applications, and it comes pre-installed on Platform.sh. The default [build flavor](/overview/build-deploy.md#building-the-application) for PHP application runs `composer install` during build, handling all of your dependencies automatically. Platform.sh is also the magic behind Symfony Cloud and Magneto Commerce.
+
+Whether Drupal 8 or legacy, Platform.sh has tried and tested best practices for Drupal deployments.
```
Comment (Crell) [inline src/languages/php/frameworks.md]: I don't think this line is necessary.

File: src/languages/php/frameworks.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
+# PHP Featured Frameworks
+
+Platform.sh is the chosen cloud solution behind Symfony, Magento, eZ Systems, and Drupal Commerce. Full Drush support and Composer-based builds make handling dependencies and builds for PHP frameworks as simple as committing your `composer.json` to your project.
+
+## Drupal
+
+Drupal 8 is built around two technologies that make developer's lives easier: Composer and Symfony. Composer simplifies dependency management for PHP applications, and it comes pre-installed on Platform.sh. The default [build flavor](/overview/build-deploy.md#building-the-application) for PHP application runs `composer install` during build, handling all of your dependencies automatically. Platform.sh is also the magic behind Symfony Cloud and Magneto Commerce.
```
Comment (Crell) [inline src/languages/php/frameworks.md]: See above; I don't think we're allowed to mention our white labels.  It's also off topic for the docs as this is not marketing copy; just the fax, sir.

File: src/languages/php/frameworks.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+* [Symfony 3](https://github.com/platformsh-templates/symfony3)
+* [Symfony 4](https://github.com/platformsh-templates/symfony4)
+
+#### Examples
+
+* [Symfony 2](https://github.com/platformsh/platformsh-example-symfony2)
+
+## TYPO3
+
+Platform.sh provides an excellent solution for hosting TYPO3 CMS projects. Set up your, database, caching, and extensions - then deploy.
```
Comment (Crell) [inline src/languages/php/frameworks.md]: Again, too markety sounding.

File: naming.md (hunk 1 above)
Comment (otaviojava) [reply naming.md]: This information is important, but need to go into `CONTRIBUTING.md`

**Pull request closed without merging**